From 0d4ba3868fa19f6ba474ee277deda650bde7e716 Mon Sep 17 00:00:00 2001
From: Konsta <konsta09@gmail.com>
Date: Sat, 17 May 2014 18:16:45 +0300
Subject: [PATCH 1/2] comment out media & gps configs

Or we'll get the wrong ones. Looks like my pretty device
tree doesn't quite behave with caf build system. :(

Change-Id: I8e62f2a3044869f92f1d8807df85260deb107831
---
 common.mk |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/common.mk b/common.mk
index d750488..54b0df9 100644
--- a/common.mk
+++ b/common.mk
@@ -167,7 +167,7 @@ FM := qcom.fmradio
 FM += libqcomfm_jni
 
 #GPS
-GPS_HARDWARE := gps.conf
+#GPS_HARDWARE := gps.conf
 GPS_HARDWARE += gps.default
 GPS_HARDWARE += gps.mahimahi
 GPS_HARDWARE += libloc_adapter
@@ -707,8 +707,8 @@ endif # BOARD_HAVE_BLUETOOTH_BLUEZ
 PRODUCT_COPY_FILES += \
     device/qcom/common/sec_config:system/etc/sec_config
 
-PRODUCT_COPY_FILES += device/qcom/common/media/media_profiles.xml:system/etc/media_profiles.xml \
-                      device/qcom/common/media/media_codecs.xml:system/etc/media_codecs.xml
+#PRODUCT_COPY_FILES += device/qcom/common/media/media_profiles.xml:system/etc/media_profiles.xml \
+#                      device/qcom/common/media/media_codecs.xml:system/etc/media_codecs.xml
 
 # enable overlays to use our version of
 # source/resources etc.
-- 
1.7.9.5

